project.clj

```diff
@@ -1,6 +1,6 @@
-(def jruby-version "9.1.11.0")
+(def jruby-version "9.1.15.0")
 
-(defproject puppetlabs/jruby-deps "9.1.11.0-2-SNAPSHOT"
+(defproject puppetlabs/jruby-deps "9.1.15.0-1-SNAPSHOT"
```

Comment:
Oh interesting, I was not aware of this versioning scheme. This is just so the project's version can mostly match jruby's?

Reply:
yea exactly. i think the `-1` here is in case we need to exclude bits of jruby (as has happened before)